solace/json_schemas/server.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "properties": {
-    "msvVpn": {
```

Comment:
Hey @195858, It would be great if you could increment the binding version as well.

Reply:
Bumped the version number to 0.2.1